src/generate.rs

```diff
@@ -254,9 +224,9 @@ pub fn generate_pair(opt: &GenerateOptions, rng: &mut impl Rng) -> (Sequence, Se
             for _ in 0..opt.length / opt.pattern_length {
                 a.append(&mut pattern.to_string().into_bytes());
             }
-            let mut aa = ropey::Rope::from_str(std::str::from_utf8(&a).unwrap());
+            let mut a_rope = ropey::Rope::from_str(std::str::from_utf8(&a).unwrap());
             for _ in 0..(opt.length as f32 * opt.error_rate / 2 as f32) as usize {
```

Comment:
instead of `2 as f32` you can simply do `2.` (here and elsewhere).

Reply:
cannot divide `f32` by `{integer}`